directjngine/src/main/java/com/softwarementors/extjs/djn/servlet/DirectJNgineServlet.java

```diff
@@ -561,6 +563,15 @@ else if( RequestRouter.isSourceRequest(pathInfo)) {
     }
   }
 
+  public static final boolean isMultipartContent(HttpServletRequest request) {
+    if (!("POST".equalsIgnoreCase(request.getMethod()) || "PUT".equalsIgnoreCase(request.getMethod()))) {
+      return false;
+    }
+    else {
```

Comment:
don't really need the else wrapper

Reply:
Simplified method: 
https://github.com/sonatype/directjngine/pull/13/commits/48188851263012bebaf095831332f3ddd6165805